app/controllers/coworking_controller.rb

```diff
@@ -51,7 +51,7 @@ def create
   def confirm_removal
     @prisoner = offender(coworking_nomis_offender_id_from_url)
 
-    @responsibility_type = ResponsibilityService.calculate_pom_responsibility(@prisoner)
+    @responsibility_type = HandoverDateService.calculate_pom_responsibility(@prisoner)
```

Comment:
Unrelated to this refactor, but a potential bug 🐛:

I'm pretty sure we should be using `@prisoner.pom_responsibility` rather than calculating it manually with `HandoverDateService.calculate_pom_responsibility` here. Otherwise overrides will not be taken into account.

Reply:
This code is a bit strange. It is trying to display the fact that the Primary POM is in a Responsible/Supporting role as we remove the co-working POM. I guess you're right.

Reply:
Fixed by #1435